code/controllers/configuration.dm

```diff
@@ -203,7 +203,7 @@ var/global/list/gamemode_cache = list()
 	var/dsay_allowed = 1
 	var/aooc_allowed = 1
 
-	var/starlight = 0	// Whether space turfs have ambient light or not
+	var/ambient_light = 0	// Whether space turfs have ambient light or not
```

Comment:
I dunno how I feel about calling this `ambient_light`, I feel like it conflates starlight and the ambient lighting system -- which is not just for starlight and the like.

Reply:
Not attached to 'starlight' specifically, just feels like some differentiation would be good.